src/main/java/core/basesyntax/service/FruitTransactionService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface FruitTransactionService {
+    void add(FruitTransaction transaction);
```

Comment:
```suggestion
    void processTransactions(List<FruitTransaction> transactions);
```

let's pass here all parsed transactions and process them